src/types.tsx

```diff
@@ -1,12 +1,33 @@
 import { TezosFilter } from 'conseiljs';
+import * as Immutable from 'immutable';
 
 export interface ArronaxState {
     filters: TezosFilter;
     dataView: DataView;
     network: string;
 }
 
-export enum DataView {Blocks, Accounts, Operations}
+export interface ImmutableState {
```

Comment:
For my understanding, can you describe a little bit how you are setting up the immutable Redux state?

Reply:
```const initialState = fromJS({
  filters: emptyFilters,	  filters: emptyFilters,
  dataView: DataView.Blocks,	  dataView: DataView.Blocks,
  network: 'zeronet'	  network: 'zeronet'
};	});```

and then on redux 
```state.set('filters', fromJS(action.filters))```